examples/webgl_loader_gltf_transmission.html

```diff
@@ -51,15 +52,15 @@
 						texture.dispose();
 						pmremGenerator.dispose();
 
-						render();
-
 						// model
 
 						const loader = new GLTFLoader().setPath( 'models/gltf/' );
-						loader.load( 'TransmissionTest.glb', function ( gltf ) {
+						loader.load( 'IridescentDishWithOlives.glb', function ( gltf ) {
 
+							mixer = new THREE.AnimationMixer( gltf.scene );
+							mixer.clipAction( gltf.animations[ 0 ] ).play();
 							scene.add( gltf.scene );
-							render();
+							animate();
```

Comment:
oops.. you already called `animate()`.

Reply:
Fixed! https://github.com/mrdoob/three.js/commit/59230b6506d87907d3ecbc520d03dfeffafc4bc1